### DISCUSSION TITLE
which time to build a hybrid demo for  winform

### DISCUSSION THREAD

**Discussion by ngyg12** [dotnetcore/BootstrapBlazor#4475] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the solution you'd like

I have build a blazor hybrid project for winform with BootStrapBlazor package.But it seems not work well. The split Component  can not fill the parent container in height side.
razor code:
` <Layout ShowFooter="true" IsPage="true" IsFixedHeader="true" IsFixedFooter="true">
    <Header>
        <div class="base-lay-header w-100">Header</div>
    </Header>
    <Main>
        <div class="d-flex w-100 h-100" style="background-color:aquamarine">
            <Split class="d-flex w-100 h-100">
                <FirstPaneTemplate>
                    <div class="d-flex h-100">我是左侧面板</div>
                </FirstPaneTemplate>
                <SecondPaneTemplate>
                    <div class="d-flex h-100">我是右侧面板</div>
                </SecondPaneTemplate>
            </Split>
        </div>
    </Main>
    <Footer>
        <div class="base-lay-footer w-100">Footer</div>
    </Footer>
</Layout>`
   

css:
.base-lay-header {
    height: 30px;
    background-color: #008000;
}
.base-lay-main {
    flex: 1;
    background-color: #E9967A;
}
.base-lay-footer {
    height: 30px;
    background-color: #F0E68C;
}
![image](https://github.com/user-attachments/assets/af8070fc-c568-4598-9594-7d62325b6d6b)

**Comment by bb-auto** [comment]
@ngyg12 Thank you for reporting. We will give triage later.
